hive/src/test/java/org/apache/iceberg/hive/legacy/TestLegacyHiveTableScan.java

```diff
@@ -182,6 +197,8 @@ private static StorageDescriptor storageDescriptor(List<FieldSchema> columns, St
         storageDescriptor.setOutputFormat("org.apache.hadoop.hive.ql.io.avro.AvroContainerOutputFormat");
         storageDescriptor.setInputFormat("org.apache.hadoop.hive.ql.io.avro.AvroContainerInputFormat");
         serDeInfo.setSerializationLib("org.apache.hadoop.hive.serde2.avro.AvroSerDe");
+        storageDescriptor.setParameters(ImmutableMap.of(
+            AvroSerdeUtils.AvroTableProperties.SCHEMA_LITERAL.getPropName(), schemaLiteral(columns)));
```

Comment:
Since we may also want to test ORC tables without avro.schema.literal, can we decouple setting of avro.schema.literal from the table format?

Reply:
We decided to handle this later